app/models/concerns/asset_summarisable.rb

```diff
@@ -18,20 +20,24 @@ def partner_savings
 
   # Investments
   def smod_investments
-    investments if investments_in_dispute
+    if any_smod_assets?
```

Comment:
It might be worth being consistent about how we do these checks - in `smod_savings` we're doing a `return unless`, here we're doing an `if` - for readability, since it's applying the same logic in every method, ideally we'd do it in the same way.